recipes/pyopcode/bld.bat

```diff
@@ -0,0 +1,46 @@
+
+set BUILD_CONFIG=Release
+
+if %PY_VER%==2.7 (
+    set GENERATOR_NAME=Visual Studio 9 2008
+)
+if %PY_VER%==3.5 (
+    set GENERATOR_NAME=Visual Studio 14 2015
+)
+
+REM pick architecture
+if %ARCH%==64 (
+	set GENERATOR_NAME=%GENERATOR_NAME% Win64
```

Comment:
Would be worth trying `%CMAKE_GENERATOR%` as it is set by `conda-build` to handle this stuff. See [here]( http://conda.pydata.org/docs/building/environment-vars.html ).